### ISSUE TITLE
Move serializers to a separate project and NuGet to support System.Text.Json in another project and NuGet

### ISSUE THREAD

**Issue by xperiandri** [GeoJSON-Net/GeoJSON.Net#148]
Create a new project for NewtonSoft.JSON serializers and issue them as a separate NuGet package

**Comment by matt-lethargic** [comment]
@xfischer I've been looking at the issues that System.Text.Json has brought about and thinking that this could be a solution in a v2 of GeoJSON.Net. 

- Natively support System.Text.Json
- Contrib lib for Json.Net

Thoughts?

**Comment by xfischer** [comment]
Yes I agree to set System.Text.Json as the default (de)serializer, and provide backwards compatibility as a separate NuGet package, as suggested by @xperiandri.
Let's create a branch for that purpose and get started. @matt-lethargic did you look how to migrate existing converters ? As I saw quickly it seems like a **lot** of work to support System.Text.Json, but hopefully we are covered with unit tests.

I'll take care of the Contribs in a similar new-branch approach.

Related MS docs : 
- https://docs.microsoft.com/dotnet/standard/serialization/system-text-json-migrate-from-newtonsoft-how-to
- https://docs.microsoft.com/dotnet/standard/serialization/system-text-json-converters-how-to

**Comment by matt-lethargic** [comment]
I haven't looked at the migration properly yet, but have docs in front of me.
Will start a v2 branch

**Comment by steveoh** [comment]
I've been migrating a similar project to STJ and it's not that bad. I was going to fork this for my project but I am glad that it is on your radar and I am happy to contribute. How's the progress so far?

**Comment by steveoh** [comment]
Since there wasn't any visible project, I went ahead and forked. Please provide some early feedback and participate!
